waveorder/stokes.py

```diff
@@ -0,0 +1,344 @@
+"""
+Overview
+--------
+
+This module collects Stokes- and Mueller-related calculations. 
+
+The functions are organized into three groups:
+
+1) A forward function group: 
+
+A = A_matrix(swing, scheme="5-State")
+s0, s1, s3, s3 = s0123_CPL_after_ADR(ret, ori, tra, dop)
+s0, s1, s2 = s012_CPL_after_AR(ret, ori, dop)
+
+2) An inverse function group:
+
+ret, ori, tra, dop = inverse_s0123_CPL_after_ADR(s0, s1, s2, s3)
+ret, ori, tra = inverse_s012_CPL_after_AR(s0, s1, s2)
+M = AR_mueller_from_CPL_projection(s0, s1, s2, s3)
```

Comment:
I suggest:
`M = mueller_from_projection(s0,s1,s2,s3, model='AR', input='CPL')`

Reply:
Much better!

Thinking more, I also don't think `projection` is the right word here. I think:

`M = mueller_from_stokes(s0, s1, s2, s3, model="AR", direction="forward", input="CPL")`

or possibly `mueller_from_measured_stokes` are even clearer.